src/containers/Header/reducer.js

```diff
@@ -1,4 +1,4 @@
-import { HELP_CONTENT, SET_BREADCRUMB_NAME } from '../../actions';
+import { SET_HELP_PATH, SET_BREADCRUMB_NAME } from '../../actions';
 
 const initialState = {
 	content: "",
```

Comment:
@aringocode Shouldnt here be some default content as we spoke of after the standup? Meaning https://docs.teskalabs.com/logman.io/ or https://docs.teskalabs.com/logman.io/user/ or so...

Btw. can you please rename `content` to `path`? Since its not content any longer, but path/url